src/web/components/tab/tab.js

```diff
@@ -47,13 +47,13 @@ const StyledDiv = styledExcludeProps(styled.div, ['active'])`
   margin-bottom: ${props => (props.active ? '-2px' : undefined)};
   border-top: ${props =>
     props.active ? '2px solid ' + Theme.green : '2px solid ' + Theme.white};
-  :hover {
+  &:hover {
     border-top: ${props =>
       props.active
-        ? '2px solid ' + Theme.white
+        ? '2px solid ' + Theme.green
```

Comment:
Is this change intended?

Reply:
Yes in order to keep the top border green when a tab is highlighted.